libs/application-generic/src/http/headers.types.ts

```diff
@@ -12,7 +12,9 @@ export enum HttpRequestHeaderKeysEnum {
   NOVU_USER_AGENT = 'Novu-User-Agent',
   BYPASS_TUNNEL_REMINDER = 'Bypass-Tunnel-Reminder',
   IDEMPOTENCY_KEY = 'Idempotency-Key',
+  // todo remove NOVU_SELF_HOSTED_TOKEN
   NOVU_SELF_HOSTED_TOKEN = 'Novu-Self-Hosted-Token',
```

Comment:
Let's remove it now?

Reply:
Haha that was the plan, I had trouble removing it at the exact time I wrote the comment, I'm not sure what it was.
I'll delete it I know, thanks for the ping!